FirebaseRemoteConfigSwift/Tests/SwiftAPI/APITestBase.swift

```diff
@@ -55,6 +84,9 @@ class APITestBase: XCTestCase {
   override func tearDown() {
     if APITests.useFakeConfig {
       fakeConsole.empty()
+    } else {
+      console.removeRemoteConfigValue(forKey: Constants.sith)
```

Comment:
Migrated from APITests.swift as part of centralizing/unifying all test setup and tear down